driver/src/price_estimation/kraken/api.rs

```diff
@@ -1,4 +1,5 @@
-use crate::http::{HttpClient, HttpFactory};
+use crate::http::{HttpClient, HttpFactory, UnlabeledHttpClient};
```

Comment:
Why are we using an UnlabeledClient here?

Reply:
Because there isn't much value in differentiating the different Kraken HTTP requests.